### PR TITLE
Technote about r-universe build system

### PR DESCRIPTION
**Pull request by jeroen**

### CODE REVIEW

**Comment by karthik** [comment]
Some comments:

> This post is part of a series of technotes about our new r-universe project. 

There isn't enough context to kick this off.
 
**Suggestion:** This post is part of a series of technotes about our new effort to hosting and deploying R packages called The r-universe project. This post is the first in a series to describe and document features and technical details of this system. 


> R-universe is a relatively complex system, consisting of many moving pieces, which combine a range of front-end, back-end, and infrastructural features, on multiple platforms.

This is a complex sentence and hard to follow. Relative to what? Consider breaking it up as it confounds multiple aspects.

**Suggestion:** The R-universe system is a complex effort, consisting of numerous frontend and backend components that are designed to work on various platforms.

> A lot of the early work on R-universe has gone into experimenting with designs to gradually build up such a system in a way that is robust and scalable,

Maybe say iterating rather than experimenting since we didn't necessarily experiment.

> Source monorepos: Manage package repositories as monorepos with a central registry.

Since this is the first mention of monorepos, maybe define it here rather than further below? Monorepos here could also link to the Wikipedia article.

> Note that the packages do not need to be hosted on GitHub themselves: they may live on any public git server, e.g. Gitlab or a university self hosted git server. The only thing that matters is that the git server is public readable, so the package source code can be checked out when initiating the submodule.

Capitalize Git (both times).


> Based on the sources from the monorepo, we arrive at the second part of the system: building all the things.

capitalize Building.

File: content/blog/2021-03-04-r-universe-buildsystem/index.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+The use of monorepos is popular in big software companies like Google and Facebook as a way to have a common state and history for the entire software suite as a whole. Our use is slightly different, because package authors do not commit directly to the monorepo, but instead we track upstream packages using submodules. But the purpose is the same: the monorepo provides a central place to track sources from the entire repository. It provides the exact state and history of packages and versions contained in the repository, simply using Git.
+
+Note that the packages do not need to be hosted on GitHub themselves: they may live on any public Git server, e.g. Gitlab or a university self hosted Git server. The only thing that matters is that the Git server is public readable, so the package source code can be checked out when initiating the submodule.
```
Comment (mpadge) [inline content/blog/2021-03-04-r-universe-buildsystem/index.md:54]: > the Git server is public readable

should be

> the Git server is publicly readable

Comment (jeroen) [reply content/blog/2021-03-04-r-universe-buildsystem/index.md:54]: thanks!
